### PR TITLE
feat: JSON nested menu editor

### PR DESCRIPTION
**Pull request by Davidmattei**
|Q              |A  |
|---------------|---|
|Bug fix?       |no|
|New feature?   |yes|	
|BC breaks?     |no|	
|Deprecations?  |no|	
|Fixed tickets  |no|

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -186,7 +192,13 @@ services:
             - { name: form.type }
     ems.fieldtype.json_menu_link:
         class: EMS\CoreBundle\Form\DataField\JsonMenuLinkFieldType
-        arguments: ['@security.authorization_checker', '@form.registry', '@ems.service.elasticsearch', '@ems.service.contenttype', '@app.elasticsearch', '@ems_common.text.decoder']
+        arguments: ['@security.authorization_checker', '@form.registry', '@ems.service.elasticsearch', '@ems.service.contenttype', '@app.elasticsearch', '@ems_common.json.decoder']
+        tags:
+            - { name: ems.form.datafieldtype, alias: choice }
+            - { name: form.type }
```
Comment (theus77) [inline Resources/config/services.yml:198]: alias:json_menu ?

File: Resources/config/services.yml [hunk 2]
```diff
@@ -186,7 +192,13 @@ services:
             - { name: form.type }
     ems.fieldtype.json_menu_link:
         class: EMS\CoreBundle\Form\DataField\JsonMenuLinkFieldType
-        arguments: ['@security.authorization_checker', '@form.registry', '@ems.service.elasticsearch', '@ems.service.contenttype', '@app.elasticsearch', '@ems_common.text.decoder']
+        arguments: ['@security.authorization_checker', '@form.registry', '@ems.service.elasticsearch', '@ems.service.contenttype', '@app.elasticsearch', '@ems_common.json.decoder']
+        tags:
+            - { name: ems.form.datafieldtype, alias: choice }
```
Comment (theus77) [inline Resources/config/services.yml:197]: alias: json_menu_link?

File: Resources/config/services.yml [hunk 3]
```diff
@@ -186,7 +192,13 @@ services:
...
+            - { name: form.type }
+    ems.fieldtype.json_menu_nested_link:
+        class: EMS\CoreBundle\Form\DataField\JsonMenuNestedLinkFieldType
+        arguments: ['@security.authorization_checker', '@form.registry', '@ems.service.elasticsearch', '@app.elasticsearch', '@ems_common.json.decoder']
         tags:
             - { name: ems.form.datafieldtype, alias: choice }
```
Comment (theus77) [inline Resources/config/services.yml:203]: alias: json_menu_nested_link?

**Review by theus77 — COMMENTED** [review]
Very impressive PR (y) 2 small remarks

File: Resources/config/services.yml (hunk 3 above)
Comment (Davidmattei) [reply Resources/config/services.yml]: https://github.com/ems-project/EMSCoreBundle/pull/405/commits/c57b1b1672c84dfb3344c3402e662cd21b4c3cf6

File: Resources/config/services.yml (hunk 2 above)
Comment (Davidmattei) [reply Resources/config/services.yml]: https://github.com/ems-project/EMSCoreBundle/pull/405/commits/c57b1b1672c84dfb3344c3402e662cd21b4c3cf6

File: Resources/config/services.yml (hunk 1 above)
Comment (Davidmattei) [reply Resources/config/services.yml:198]: https://github.com/ems-project/EMSCoreBundle/pull/405/commits/c57b1b1672c84dfb3344c3402e662cd21b4c3cf6

**Pull request merged**